awot/dropsondes/dropsondes.py

```diff
@@ -8,651 +8,673 @@
 from skew import SkewXTick
 
 
-class DropSondes(object):
```

Comment:
Got rid of the class structure and opted for functions to match latest design. Still missing a few moving parts.